tests/liberty/unit_delay.ys

```diff
@@ -0,0 +1,3 @@
+# Nothing gets imported: the file lacks timing data
+read_liberty -wb -unit_delay normal.lib
+select -assert-none =*/t:$specify*
```

Comment:
This is an almost empty test since we don't have any `.lib` files checked in with populated timing sections. We can find a new one provided the licensing is ok